### ISSUE TITLE
Packaging with Python lacks numpy

### ISSUE THREAD

**Issue by martinfalk** [inviwo/inviwo#246]
When creating a installation package (zip/installer) and the python module is enabled, the zip file will include a python executable and dlls downloaded from python.org. However, the version shipped is lacking the numpy module.

Shipped:
```
Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 17:00:18) [MSC v.1900 64 bit (AMD64)] on win32
```
But I am actually using anaconda python on the machine where I packaged Inviwo.

**Comment by martinfalk** [comment]
Maybe it is possible to run a post-installation script in the installer
1) ensure that pip exists: `bin/python -m ensurepip` or download `get-pip.py` and run `python get-pip.py`
2) run `pip install numpy`

https://stackoverflow.com/questions/40893602/how-to-install-numpy-for-python-3-6

**Comment by petersteneteg** [comment]
Might be possible to do from within inviwo on running python... needs quite a bit of testing though...

**Comment by r-englund** [comment]
this was fixed in 60584452a8b5b53491c1112ffc3230e1c90002fa
